docs/ftldns/blockingmode.md

```diff
@@ -61,7 +58,7 @@ doubleclick.net.        2       IN      A       192.168.1.42
 
 ## Pi-hole's full IP blocking mode
 
-In `IP` mode, blocked queries will be answered with the local IP addresses of your Pi-hole (see [BLOCK_IP4](configfile.md#block_ipv4) and [BLOCK_IP6](configfile.md#block_ipv6) for additional options). To set this mode explicitly, set `BLOCKINGMODE=IP` in `/etc/pihole/pihole-FTL.conf`.
```

Comment:
A this reference is removed, it should say something about `reply.blocking.IPv4` and `reply.blocking.IPv6`